deploy/templates/components/rbac/role.yaml

```diff
@@ -27,6 +27,14 @@ rules:
   - serviceaccounts
   verbs:
   - '*'
+- apiGroups:
```

Comment:
This cluster role modification was required to prevent the following error:
`E0811 17:20:17.422655 1392784 reflector.go:138] pkg/mod/k8s.io/client-go@v0.21.3/tools/cache/reflector.go:167: Failed to watch *v1.StorageClass: unknown (get storageclasses.storage.k8s.io)`